content/700-optimize/200-getting-started.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: 'Getting started'
+metaTitle: 'Getting started with Prisma Optimize'
+metaDescription: 'Learn how to get up and running with Prisma Optimize.'
+tocDepth: 3
+toc: true
+---
+
+## Prerequisites
+
+To get started with Optimize, you will need the following:
+
+- A [Prisma Data Platform workspace](https://console.prisma.io).
+- A project that uses [Prisma Client](/orm/prisma-client) `5.16.1` or higher (we always recommend using the latest version of Prisma).
+- A PostgreSQL, MySQL/MariaDB, PlanetScale, CockroachDB, or MongoDB database.
+
+## 1. Enable Optimize
+
+Login to your [Prisma Data Platform account](https://console.prisma.io) and then navigate to the [Optimize dashboard](https://optimize.prisma.io).
```

Comment:
Have we seen this?

Reply:
Should we mention that users can click on "Optimize" Tab in the sidebar? That should redirect them to https://optimize.prisma.io

Reply:
@nurul3101 @mhessdev this was correct and the required step before. It's not applicable now 🥲, so I'll be changing it.